framework/wazuh/core/cluster/tests/test_cluster.py

```diff
@@ -362,23 +363,20 @@ def test_compare_files(wazuh_db_query_mock, mock_get_cluster_items):
         files, count = cluster.compare_files(seq, condition, 'worker1')
         assert count["missing"] == 1
         assert count["extra"] == 0
-        assert count["extra_valid"] == 1
         assert count["shared"] == 1
 
     # Second condition
     condition = {'some/path5/': {'cluster_item_key': 'key', 'md5': 'md5 def value'},
                  'some/path4/': {'cluster_item_key': "key", 'md5': 'md5 value'},
-                 os.path.relpath(common.groups_path, common.wazuh_path): {'cluster_item_key': "key",
-                                                                          'md5': 'md5 value'}}
+                 'PATH': {'cluster_item_key': "key", 'md5': 'md5 value'}}
 
     files, count = cluster.compare_files(seq, condition, 'worker1')
     assert count["missing"] == 2
     assert count["extra"] == 0
-    assert count["extra_valid"] == 3
     assert count["shared"] == 0
-    wazuh_db_query_mock.assert_called_once_with(select=['id'], limit=None,
-                                                filters={'rbac_ids': ['agent-groups']},
-                                                rbac_negate=False)
+    # wazuh_db_query_mock.assert_called_once_with(select=['id'], limit=None,
+    #                                             filters={'rbac_ids': ['PATH']},
+    #                                             rbac_negate=False)

```

Comment:
If we isolate the functionality, this does not need to be commented on. The idea is that we keep the functionality but do not use it, which implies that it has to have its tests.